test/retry-only.js

```diff
@@ -0,0 +1,44 @@
+const setup = require('../index.js')
```

Comment:
Just wondering what the reason is for new test files for `retry.only` but not other functions?

Reply:
Maybe we could split the test file then, it's getting bigger and bigger. I isolated `retry.only` so it doesn't affect other tests